### ISSUE TITLE
smaller distance for empty route for if, switch etc.

### ISSUE THREAD

**Issue by xieofxie** [microsoft/BotFramework-Composer#2657]
## Is your feature request related to a problem? Please describe.

I'm always frustrated when it takes a lot of space without any element:
![image](https://user-images.githubusercontent.com/2876650/79311590-402bee00-7f30-11ea-8f10-fa09539b5919.png)

## Describe the solution you'd like

Yellow space should follow red one's distance

**Comment by yeze322** [comment]
@DesignPolice Hey Marc, do you know what's our expected pixel margin in design?

**Comment by DesignPolice** [comment]
Wow. always good to see what these can look like in real use... I had not planned for loops getting that deep. Let me get you some numbers or maybe just so gap limits. Thanks @yeze322

**Comment by DesignPolice** [comment]
@yeze322 I have looked at this a bit. I think our magic number is going to be 32px.

I made a comp, I think if we leave 32 on each side of a line and nodes (not counting the + ) we should have a easy setting - there will be some strange looking spots if they have loops that go from the top to the bottom hopefully not to many. I think we can continue to collect use cases and see what is happening. 

![Screen Shot 2020-04-15 at 9 49 19 AM](https://user-images.githubusercontent.com/45223620/79364956-de06c500-7efe-11ea-946d-f819704cfea8.png)
